src/css/components/badge.css

```diff
@@ -0,0 +1,57 @@
+.badge-container {
+	display: inline-block;
+	position: relative;
+	font-size: 2rem;
+}
+
+.badge-sm {
```

Comment:
Make a class of badge and give the basic styling and now add badge sm class after that by reducing the size

Reply:
Addressed it!, thanks!!